<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1180</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1180</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>12526</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>991</width>
      <height>562</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="maximumSize">
        <size>
         <width>900</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Всего элементов в базе: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="maximumSize">
        <size>
         <width>900</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Всего элементов категории 'Электроника' : 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>900</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>900</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Добавить элемент</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>904</width>
      <height>665</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>150</height>
        </size>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
       <item>
        <property name="text">
         <string>Электроника</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Мебель</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Одежда</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Компьюьерные комплектующие</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Телефоны</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Игрушки</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>60</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>Двойное нажатие 
на  элемент 
в таблице - удаление</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Выбрать категорию</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Сохранить как ...</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Создать новый</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Редактировать выбранный элемент</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Удать выбранный элемент</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
